tasks/internal/push-artifacts-to-cdn-task/tests/mocks.sh

```diff
@@ -65,6 +65,7 @@ function ziputil() {
     fi
 
     if [[ "$1" =~ (r|czf)$ ]]; then
+        set -x
```

Comment:
Why add this? I see you added `set -x` to the actual step as well, but the mocks.sh file has `set -x` at the top as well. Was it turned off somewhere along the way? As far as I can see, all the places that turn it off will turn it back on a few lines later.